src/settings/slider-setting/slider-setting.ts

```diff
@@ -1,70 +1,80 @@
   /** Maximum range value. */
-  @attr({defaultValue: '100'}) public max: string;
+  @attr({defaultValue: '0'}) public max: string;
   /** Step for range. */
-  @attr({defaultValue: '1'}) public step: string;
-  /** Range input to change setting's value. */
-  protected $field: HTMLInputElement;
-  /** Range value displayed below. */
-  protected $fieldValue: HTMLDivElement;
+  @attr({defaultValue: '0'}) public step: string;
 
   protected connectedCallback() {
     super.connectedCallback();
-    if (this.$field) return;
 
-    this.initSlider();
-    this.initSliderValue();
-    const label = document.createElement('label');
-    label.innerText = this.label;
-    label.appendChild(this.$field);
-
-    this.innerHTML = '';
-    this.append(label);
+    this.updateConfiguration();
+    this.append(this.$label);
     this.append(this.$fieldValue);
   }
 
-  protected initSlider() {
-    this.$field = document.createElement('input');
-    this.$field.type = 'range';
-    this.$field.min = this.min;
-    this.$field.max = this.max;
-    this.$field.step = this.step;
-    this.$field.name = this.label;
+  @memoize()
+  protected get $field(): HTMLInputElement {
+    const $field = document.createElement('input');
+    $field.type = 'range';
+
+    $field.min = this.min;
+    $field.max = this.max;
+    $field.step = this.step;
+
+    return $field;
   }
 
-  protected initSliderValue() {
-    this.$fieldValue = document.createElement('div');
-    this.$fieldValue.classList.add('slider-value');
-    this.$field.addEventListener('input', () => this.updateSliderValue());
+  @memoize()
+  protected get $label(): HTMLLabelElement {
+    const $label = document.createElement('label');
+    $label.innerHTML = `<span>${this.label}</span>`;
+    return $label;
   }
 
-  protected updateSliderValue() {
-    this.$fieldValue.textContent = `Value: ${this.$field.value}`;
+  protected updateConfiguration() {
+    this.$field.addEventListener('input', this.updateSliderValue);
+    this.$label.append(this.$field);
+  }
+
+  @memoize()
+  protected get $fieldValue(): HTMLDivElement {
+    const $fieldValue = document.createElement('div');
+    $fieldValue.classList.add('slider-value');
+    return $fieldValue;
   }
 
+  protected updateSliderValue = () => {
+    this.$fieldValue.textContent = `Value: ${this.$field.value}`;
+  };
+
   protected getDisplayedValue(): string {
     return this.$field.value;
   }
 
   protected setValue(value: string | null): void {
-    this.$field.value = value || '0';
+    this.$field.value = value || this.min;
     this.updateSliderValue();
   }
 
   protected setInconsistency(msg = WARNING_MSG.inconsistent): void {
-    this.$field.value = '0';
+    this.$field.value = this.min;
     this.$fieldValue.textContent = msg;
   }
+
+  protected disconnectedCallback() {
+    this.$field.removeEventListener('input', this.updateSliderValue);
+    this.innerHTML = '';
+    super.disconnectedCallback();
+  }
```

Comment:
Super cosmetic: place it near connected callback for consistency with other components.